contents/stats.md

```diff
@@ -0,0 +1,430 @@
+
+**Inferential statistics allows generating inferences** (statements) **from observed data about the data generation process**.
+All phenomena have an underlying data generating process that describes how the data is being generated.
+For example, in a soccer game, a scored goal can be explained by an underlying process: a tactic, error, struck of luck; or a mix of those.
+If we know a phenomenon's data generating process, we can use probability to simulate possible scenarios given certain aspects.
+Most of the time, especially in applied sciences, we do not have full knowledge of the data generating process.
+Given the observed data, we can retrace our way back to the data generating process.
+This process is known as **statistical inference**.
+Given some data, we can *infer* what are the aspects of the underlying data generating process.
+This is the realm of inferential statistics.
+
+In @fig:stats_vs_prob we summarize the relationship between data generating process and observed data.
+With knowledge of the data generating process we can apply probability to generate and simulate plausible data.
+And by using the observed data we can use inference to gain knowledge about the underlying data generating process.
+
+```jl
+fig = statistics_graph()
+caption = "Statistics vs Probability"
+label = "stats_vs_prob"
+Options(fig; filename=label, caption, label)
+```
+
+In this chapter, we will cover only descriptive statistics.
+Inferential statistics is an important and fundamental component of applied sciences, but its scope is too broad. Let's begin on simple ways to summarize our data with central tendencies.
+
+## Central Tendencies Measures {#sec:stats_central}
+
+The most basic way of using descriptive statistics is to summarize data by a measure of **central tendency**.
+
+### Mean {#sec:stats_central_mean}
+
+The most common central tendency measure is the mean.
+**The mean, also known as average, is the sum of all measurements divided by the number of observations** and we generally denote it as "x bar":
+
+$$ \bar{x} = \frac{1}{n} \sum^n_{i=1} x_i = \frac{x_1 + x_2 + \cdots + x_n}{n}, $$ {#eq:mean}
+
+where $\bar{x}$ is the sample mean of the variable $\mathbf{x} = x_1, \cdots, x_n$.
+Often, we see the mean denoted with the **Greek letter $\mu$**, for example $\bar{x} = \mu_x$.
+Additionally, the mean is also known as the **expectation** which is represented by the operator $\operatorname{E}$, thus the mean of a variable $x$ becomes a $\operatorname{E}(x)$[^expectation].
+So, bear in mind that you might find different notations for the mean.
+
+[^expectation]: technically speaking, the expectation is defined as either a sum for discrete variables or an integral for continuous variables. In contrast with the mean which is the sum of the variables' values divided by number of values, the expectation can be defined over an infinite domain while the mean only works for finite domains.
```

Comment:
```suggestion
[^expectation]: technically speaking, the expectation is defined as either a sum for discrete variables or an integral for continuous variables weighted by a probability distribution. In contrast with the mean which is simply the arithmetic sum of the variables' values divided by number of values.
```
I would leave it like this. Regarding domains and existence, finiteness,  of the mean depends on the probability distribution, so better not go into much details here.

Reply:
Thanks! Works great!